### PR TITLE
Docs update on data connectors & data accelerators

### PR DESCRIPTION
**Pull request by Sevenannn**
## 🗣 Description

Standardized data connector documentation. Added cookbook links to all data connector & accelerator docs.

### CODE REVIEW

File: spiceaidocs/docs/components/data-connectors/github.md [hunk 1]
```diff
@@ -143,7 +159,7 @@ datasets:
 #### Schema
 
 | Column Name     | Data Type    | Is Nullable |
-|-----------------|--------------|-------------|
+| --------------- | ------------ | ----------- |
```
Comment (lukekim) [inline spiceaidocs/docs/components/data-connectors/github.md]: This looks wrong?

File: spiceaidocs/docs/components/data-accelerators/arrow.md [hunk 2]
```diff
@@ -47,3 +47,7 @@ When accelerating a dataset using the In-Memory Arrow Data Accelerator, some or
 In-memory limitations can be mitigated by storing acceleration data on disk, which is supported by [`duckdb`](./duckdb.md) and [`sqlite`](./sqlite.md) accelerators by specifying `mode: file`.
 
 :::
+
+## Cookbook
+
+- A cookbook recipe to configure In-Memory Arrow as data accelerator in Spice. [In-Memory Arrow Data Accelerator Recipe](https://github.com/spiceai/cookbook/tree/trunk/arrow#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-accelerators/arrow.md]:
```suggestion
- A cookbook recipe to configure In-Memory Arrow as data accelerator in Spice. [In-Memory Arrow Data Accelerator](https://github.com/spiceai/cookbook/tree/trunk/arrow#readme)
```

File: spiceaidocs/docs/components/data-accelerators/data-refresh.md [hunk 3]
```diff
@@ -446,7 +446,13 @@ This acceleration configuration applies a number of different behaviors:
...
 6. Running a query to retrieve logs within a range of now to longer than 1 day ago will only return logs from the last day. This is due to the `refresh_data_window` only accelerating the last day's worth of logs, which will return some results. Because results are returned, Spice will not fall back to the source even though `on_zero_results: use_source` is specified.
 7. Spice will retain newly appended log rows for 7 days before discarding them, as specified by the `retention_*` parameters.
+
+## Cookbook
+
+- Configure accelerated dataset retention policy. [Accelerated Dataset Retention Policy Recipe](https://github.com/spiceai/cookbook/tree/trunk/retention#readme)
+- Dynamically refresh specific data at runtime by programmatically updating refresh_sql and triggering data refreshes. [Advanced Data Refresh Recipe](https://github.com/spiceai/cookbook/tree/trunk/acceleration/data-refresh#readme)
+- Configure `refresh_data_window` to filter refreshed data to recent data [Refresh Data Window Recipe](https://github.com/spiceai/cookbook/tree/trunk/refresh-data-window#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-accelerators/data-refresh.md]:
```suggestion
- Configure accelerated dataset retention policy. [Accelerated Dataset Retention Policy](https://github.com/spiceai/cookbook/tree/trunk/retention#readme)
- Dynamically refresh specific data at runtime by programmatically updating refresh_sql and triggering data refreshes. [Advanced Data Refresh](https://github.com/spiceai/cookbook/tree/trunk/acceleration/data-refresh#readme)
- Configure `refresh_data_window` to filter refreshed data to recent data [Refresh Data Window](https://github.com/spiceai/cookbook/tree/trunk/refresh-data-window#readme)
```

File: spiceaidocs/docs/components/data-accelerators/duckdb.md [hunk 4]
```diff
@@ -57,3 +57,7 @@ When accelerating a dataset using `mode: memory` (the default), some or all of t
 In-memory limitations can be mitigated by storing acceleration data on disk, which is supported by [`duckdb`](./duckdb.md) and [`sqlite`](./sqlite.md) accelerators by specifying `mode: file`.
 
 :::
+
+## Cookbook
+
+- A cookbook recipe to configure DuckDB as a data accelerator in Spice. [DuckDB Data Accelerator Recipe](https://github.com/spiceai/cookbook/tree/trunk/duckdb/accelerator#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-accelerators/duckdb.md]:
```suggestion
- A cookbook recipe to configure DuckDB as a data accelerator in Spice. [DuckDB Data Accelerator](https://github.com/spiceai/cookbook/tree/trunk/duckdb/accelerator#readme)
```

File: spiceaidocs/docs/components/data-accelerators/postgres/index.md [hunk 5]
```diff
@@ -110,3 +110,7 @@ The table below lists the supported [Apache Arrow data types](https://arrow.apac
 | `Duration`                             | `BigInteger`            | `bigint`                      |
 | `List` / `LargeList` / `FixedSizeList` | `Array`                 | `array`                       |
 | `Struct`                               | `N/A`                   | `Composite` (Custom type)     |
+
+## Cookbook
+
+- A cookbook recipe to configure PostgreSQL as a data accelerator in Spice. [PostgreSQL Data Accelerator Recipe](https://github.com/spiceai/cookbook/tree/trunk/postgres/accelerator#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-accelerators/postgres/index.md]:
```suggestion
- A cookbook recipe to configure PostgreSQL as a data accelerator in Spice. [PostgreSQL Data Accelerator](https://github.com/spiceai/cookbook/tree/trunk/postgres/accelerator#readme)
```

File: spiceaidocs/docs/components/data-accelerators/sqlite.md [hunk 6]
```diff
@@ -54,3 +54,7 @@ When accelerating a dataset using `mode: memory` (the default), some or all of t
 In-memory limitations can be mitigated by storing acceleration data on disk, which is supported by [`duckdb`](./duckdb.md) and [`sqlite`](./sqlite.md) accelerators by specifying `mode: file`.
 
 :::
+
+## Cookbook
+
+- A cookbook recipe to configure SQLite as a data accelerator in Spice. [SQLite Data Accelerator Recipe](https://github.com/spiceai/cookbook/tree/trunk/sqlite/accelerator#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-accelerators/sqlite.md]:
```suggestion
- A cookbook recipe to configure SQLite as a data accelerator in Spice. [SQLite Data Accelerator](https://github.com/spiceai/cookbook/tree/trunk/sqlite/accelerator#readme)
```

File: spiceaidocs/docs/components/data-connectors/clickhouse.md [hunk 7]
```diff
@@ -100,3 +100,7 @@ datasets:
 ## Secrets
 
 Spice integrates with multiple secret stores to help manage sensitive data securely. For detailed information on supported secret stores, refer to the [secret stores documentation](/components/secret-stores). Additionally, learn how to use referenced secrets in component parameters by visiting the [using referenced secrets guide](/components/secret-stores#using-secrets).
+
+## Cookbook
+
+- A cookbook recipe to configure ClickHouse as data connector in Spice. [Clickhouse Data Connector Recipe](https://github.com/spiceai/cookbook/tree/trunk/clickhouse#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/clickhouse.md]:
```suggestion
- A cookbook recipe to configure ClickHouse as data connector in Spice. [Clickhouse Data Connector](https://github.com/spiceai/cookbook/tree/trunk/clickhouse#readme)
```

File: spiceaidocs/docs/components/data-connectors/databricks.md [hunk 8]
```diff
@@ -191,3 +191,7 @@ The table below shows the Databricks (mode: delta_lake) data types supported, al
 ## Secrets
 
 Spice integrates with multiple secret stores to help manage sensitive data securely. For detailed information on supported secret stores, refer to the [secret stores documentation](/components/secret-stores). Additionally, learn how to use referenced secrets in component parameters by visiting the [using referenced secrets guide](/components/secret-stores#using-secrets).
+
+## Cookbook
+
+- A cookbook recipe to configure Databricks as data connector in Spice under `delta_lake` mode. [Spice on Databricks (mode: delta_lake) Recipe](https://github.com/spiceai/cookbook/tree/trunk/databricks/delta_lake#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/databricks.md]:
```suggestion
- A cookbook recipe to configure Databricks as data connector in Spice under `delta_lake` mode. [Spice on Databricks (mode: delta_lake)](https://github.com/spiceai/cookbook/tree/trunk/databricks/delta_lake#readme)
```

File: spiceaidocs/docs/components/data-connectors/debezium.md [hunk 9]
```diff
@@ -90,12 +90,12 @@ The following settings are required:
...
+## Secrets
 
-See an example of configuring a dataset to use CDC with Debezium by following the Cookbook Recipe [Streaming changes in real-time with Debezium CDC](https://github.com/spiceai/cookbook/tree/trunk/cdc-debezium#readme).
+Spice integrates with multiple secret stores to help manage sensitive data securely. For detailed information on supported secret stores, refer to the [secret stores documentation](/components/secret-stores). Additionally, learn how to use referenced secrets in component parameters by visiting the [using referenced secrets guide](/components/secret-stores#using-secrets).
 
-An recipe of configuring [SASL authentication over SSL](https://github.com/spiceai/cookbook/tree/trunk/cdc-debezium/sasl-scram#readme) is available as well.
+## Cookbook
 
-## Secrets
+- See an example of configuring a dataset to use CDC with Debezium by following the sample [Recipe for Streaming changes in real-time with Debezium CDC](https://github.com/spiceai/cookbook/tree/trunk/cdc-debezium#readme).
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/debezium.md]:
```suggestion
- See an example of configuring a dataset to use CDC with Debezium by following the sample [Streaming changes in real-time with Debezium CDC](https://github.com/spiceai/cookbook/tree/trunk/cdc-debezium#readme).
```

File: spiceaidocs/docs/components/data-connectors/debezium.md [hunk 10]
```diff
@@ -90,12 +90,12 @@ The following settings are required:
...
 
-Spice integrates with multiple secret stores to help manage sensitive data securely. For detailed information on supported secret stores, refer to the [secret stores documentation](/components/secret-stores). Additionally, learn how to use referenced secrets in component parameters by visiting the [using referenced secrets guide](/components/secret-stores#using-secrets).
+- An example of configuring [Recipe for Streaming changes in real-time with Debezium CDC and SASL/SCRAM authentication](https://github.com/spiceai/cookbook/tree/trunk/cdc-debezium/sasl-scram#readme) is available as well.
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/debezium.md]:
```suggestion
- An example of [Streaming changes in real-time with Debezium CDC and SASL/SCRAM authentication](https://github.com/spiceai/cookbook/tree/trunk/cdc-debezium/sasl-scram#readme) is available as well.
```

File: spiceaidocs/docs/components/data-connectors/dremio.md [hunk 11]
```diff
@@ -78,3 +78,7 @@ SELECT COUNT(*) FROM cool_dataset;
 ## Secrets
 
 Spice integrates with multiple secret stores to help manage sensitive data securely. For detailed information on supported secret stores, refer to the [secret stores documentation](/components/secret-stores). Additionally, learn how to use referenced secrets in component parameters by visiting the [using referenced secrets guide](/components/secret-stores#using-secrets).
+
+## Cookbook
+
+- A cookbook recipe to configure Dremio as data connector in Spice. [Dremio Data Connector Recipe](https://github.com/spiceai/cookbook/tree/trunk/dremio#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/dremio.md]:
```suggestion
- A cookbook recipe to configure Dremio as data connector in Spice. [Dremio Data Connector](https://github.com/spiceai/cookbook/tree/trunk/dremio#readme)
```

File: spiceaidocs/docs/components/data-connectors/duckdb.md [hunk 12]
```diff
@@ -138,3 +138,7 @@ SELECT * FROM read_json('todos.json');
 - The DuckDB connector does not support `Decimal256` (76 digits), as it exceeds DuckDB's maximum Decimal width of 38 digits.
 
 :::
+
+## Cookbook
+
+- A cookbook recipe to configure DuckDB as a data connector in Spice. [DuckDB Data Connector Recipe](https://github.com/spiceai/cookbook/tree/trunk/duckdb/connector#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/duckdb.md]:
```suggestion
- A cookbook recipe to configure DuckDB as a data connector in Spice. [DuckDB Data Connector](https://github.com/spiceai/cookbook/tree/trunk/duckdb/connector#readme)
```

File: spiceaidocs/docs/components/data-connectors/github.md [hunk 13]
```diff
@@ -392,3 +408,7 @@ sql> select starred_at, login from spiceai.stargazers order by starred_at DESC l
 
 Time: 0.0088075 seconds. 10 rows.
 ```
+
+## Cookbook
+
+- A cookbook recipe to configure Github as a data connector in Spice. [GitHub Data Connector Recipe](https://github.com/spiceai/cookbook/tree/trunk/github#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/github.md]:
```suggestion
- A cookbook recipe to configure Github as a data connector in Spice. [GitHub Data Connector](https://github.com/spiceai/cookbook/tree/trunk/github#readme)
```

File: spiceaidocs/docs/components/data-connectors/graphql.md [hunk 14]
```diff
@@ -308,3 +319,7 @@ params:
       }
     }
 ```
+
+## Cookbook
+
+- A cookbook recipe to configure GraphQL as a data connector in Spice. [GraphQL Data Connector Recipe](https://github.com/spiceai/cookbook/tree/trunk/graphql#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/graphql.md]:
```suggestion
- A cookbook recipe to configure GraphQL as a data connector in Spice. [GraphQL Data Connector](https://github.com/spiceai/cookbook/tree/trunk/graphql#readme)
```

File: spiceaidocs/docs/components/data-connectors/localpod.md [hunk 15]
```diff
@@ -23,17 +23,20 @@ When synchronization is enabled, the following logs will be emitted:
...
+      refresh_check_interval: 10s
+  - from: localpod:test
+    name: test_local
+    acceleration:
+      enabled: true # This dataset accelerates the parent `test` dataset into in-memory Arrow records and is synchronized with the parent
 ```
+
+## Cookbook
+
+- A cookbook recipe to configure Localpod as a data connector in Spice. [Local dataset replication (Localpod) recipe](https://github.com/spiceai/cookbook/tree/trunk/localpod#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/localpod.md]:
```suggestion
- A cookbook recipe to configure Localpod as a data connector in Spice. [Local dataset replication (Localpod)](https://github.com/spiceai/cookbook/tree/trunk/localpod#readme)
```

File: spiceaidocs/docs/components/data-connectors/memory.md [hunk 16]
```diff
@@ -6,21 +6,26 @@ pagination_prev: null
...
+          - from: all-MiniLM-L6-v2
 
 embeddings:
   - name: all-MiniLM-L6-v2
     from: huggingface:huggingface.co/sentence-transformers/all-MiniLM-L6-v2
 ```
+
+## Cookbook
+
+- A cookbook recipe to provide persistent memory capabilities for language models in Spice. [LLM Memory Recipe](https://github.com/spiceai/cookbook/tree/trunk/llm-memory#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/memory.md]:
```suggestion
- A cookbook recipe to provide persistent memory capabilities for language models in Spice. [LLM Memory](https://github.com/spiceai/cookbook/tree/trunk/llm-memory#readme)
```

File: spiceaidocs/docs/components/data-connectors/mssql.md [hunk 17]
```diff
@@ -86,4 +86,8 @@ datasets:
 
 ## Secrets
 
-Spice integrates with multiple secret stores to help manage sensitive data securely. For detailed information on supported secret stores, refer to the [secret stores documentation](/components/secret-stores). Additionally, learn how to use referenced secrets in component parameters by visiting the [using referenced secrets guide](/components/secret-stores#using-secrets).
\ No newline at end of file
+Spice integrates with multiple secret stores to help manage sensitive data securely. For detailed information on supported secret stores, refer to the [secret stores documentation](/components/secret-stores). Additionally, learn how to use referenced secrets in component parameters by visiting the [using referenced secrets guide](/components/secret-stores#using-secrets).
+
+## Cookbook
+
+- A cookbook recipe to configure Microsoft SQL Server as a data connector in Spice. [MSSQL (Microsoft SQL Server) Connector Recipe](https://github.com/spiceai/cookbook/tree/trunk/mssql#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/mssql.md]:
```suggestion
- A cookbook recipe to configure Microsoft SQL Server as a data connector in Spice. [MSSQL (Microsoft SQL Server) Connector](https://github.com/spiceai/cookbook/tree/trunk/mssql#readme)
```

File: spiceaidocs/docs/components/data-connectors/mysql.md [hunk 18]
```diff
@@ -185,3 +185,9 @@ datasets:
 ## Secrets
 
 Spice integrates with multiple secret stores to help manage sensitive data securely. For detailed information on supported secret stores, refer to the [secret stores documentation](/components/secret-stores). Additionally, learn how to use referenced secrets in component parameters by visiting the [using referenced secrets guide](/components/secret-stores#using-secrets).
+
+## Cookbook
+
+- A cookbook recipe to configure MySQL as a data connector in Spice. [MySQL Data Connector Recipe](https://github.com/spiceai/cookbook/tree/trunk/mysql/connector#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/mysql.md]:
```suggestion
- A cookbook recipe to configure MySQL as a data connector in Spice. [MySQL Data Connector](https://github.com/spiceai/cookbook/tree/trunk/mysql/connector#readme)
```

File: spiceaidocs/docs/components/data-connectors/mysql.md [hunk 19]
```diff
@@ -185,3 +185,9 @@ datasets:
...
+- A cookbook recipe to configure AWS RDS Aurora (MySQL Compatible) as a data connector in Spice. [AWS RDS Aurora (MySQL Data Connector) Recipe](https://github.com/spiceai/cookbook/tree/trunk/mysql/rds-aurora#readme)
+- A cookbook recipe to configure Planetscale as a data connector in Spice. [Planetscale (MySQL Data Connector) Recipe](https://github.com/spiceai/cookbook/tree/trunk/mysql/planetscale#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/mysql.md]:
```suggestion
- A cookbook recipe to configure AWS RDS Aurora (MySQL Compatible) as a data connector in Spice. [AWS RDS Aurora (MySQL Data Connector)](https://github.com/spiceai/cookbook/tree/trunk/mysql/rds-aurora#readme)
- A cookbook recipe to configure Planetscale as a data connector in Spice. [Planetscale (MySQL Data Connector)](https://github.com/spiceai/cookbook/tree/trunk/mysql/planetscale#readme)
```

File: spiceaidocs/docs/components/data-connectors/odbc.md [hunk 20]
```diff
@@ -320,14 +319,18 @@ version: v1beta1
...
 
 See the [ODBC Cookbook Recipe](https://github.com/spiceai/cookbook/blob/trunk/odbc/README.md) for more help on getting started with ODBC and Postgres.
 
 ## Secrets
 
 Spice integrates with multiple secret stores to help manage sensitive data securely. For detailed information on supported secret stores, refer to the [secret stores documentation](/components/secret-stores). Additionally, learn how to use referenced secrets in component parameters by visiting the [using referenced secrets guide](/components/secret-stores#using-secrets).
+
+## Cookbook
+
+- A cookbook recipe to configure ODBC as a data connector in Spice. [ODBC Data Connector Recipe](https://github.com/spiceai/cookbook/tree/trunk/odbc#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/odbc.md]:
```suggestion
- A cookbook recipe to configure ODBC as a data connector in Spice. [ODBC Data Connector](https://github.com/spiceai/cookbook/tree/trunk/odbc#readme)
```

File: spiceaidocs/docs/components/data-connectors/postgres/index.md [hunk 21]
```diff
@@ -177,4 +175,10 @@ datasets:
...
\ No newline at end of file
+Spice integrates with multiple secret stores to help manage sensitive data securely. For detailed information on supported secret stores, refer to the [secret stores documentation](/components/secret-stores). Additionally, learn how to use referenced secrets in component parameters by visiting the [using referenced secrets guide](/components/secret-stores#using-secrets).
+
+## Cookbook
+
+- A cookbook recipe to configure PostgreSQL as a data connector in Spice. [PostgreSQL Data Accelerator Recipe](https://github.com/spiceai/cookbook/tree/trunk/postgres/accelerator#readme)
+- A cookbook recipe to configure AWS RDS for PostgreSQL as a data connector in Spice. [AWS RDS for PostgreSQL Recipe](https://github.com/spiceai/cookbook/tree/trunk/postgres/rds#readme)
+- A cookbook recipe to configure Supabase a data connector in Spice. [Supabase (PostgreSQL Data Connector) Recipe](https://github.com/spiceai/cookbook/tree/trunk/postgres/supabase#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/postgres/index.md]:
```suggestion
- A cookbook recipe to configure PostgreSQL as a data connector in Spice. [PostgreSQL Data Accelerator](https://github.com/spiceai/cookbook/tree/trunk/postgres/accelerator#readme)
- A cookbook recipe to configure AWS RDS for PostgreSQL as a data connector in Spice. [AWS RDS for PostgreSQL](https://github.com/spiceai/cookbook/tree/trunk/postgres/rds#readme)
- A cookbook recipe to configure Supabase a data connector in Spice. [Supabase (PostgreSQL Data Connector)](https://github.com/spiceai/cookbook/tree/trunk/postgres/supabase#readme)
```

File: spiceaidocs/docs/components/data-connectors/s3.md [hunk 22]
```diff
@@ -146,10 +146,8 @@ datasets:
...
-Spice supports three types of [secret stores](/components/secret-stores):
+Spice integrates with multiple secret stores to help manage sensitive data securely. For detailed information on supported secret stores, refer to the [secret stores documentation](/components/secret-stores). Additionally, learn how to use referenced secrets in component parameters by visiting the [using referenced secrets guide](/components/secret-stores#using-secrets).
 
-- [Environment variables](/components/secret-stores/env)
-- [Kubernetes Secret Store](/components/secret-stores/kubernetes)
-- [Keyring Secret Store](/components/secret-stores/keyring)
+## Cookbook
 
-Explore the different options to manage sensitive data securely.
+- A cookbook recipe to configure S3 as a data connector in Spice. [S3 Data Connector Recipe](https://github.com/spiceai/cookbook/tree/trunk/s3#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/s3.md]:
```suggestion
- A cookbook recipe to configure S3 as a data connector in Spice. [S3 Data Connector](https://github.com/spiceai/cookbook/tree/trunk/s3#readme)
```

File: spiceaidocs/docs/components/data-connectors/sharepoint.md [hunk 23]
```diff
@@ -140,3 +140,11 @@ And set the `SPICE_SHAREPOINT_BEARER_TOKEN` secret via:
...
 spice login sharepoint --tenant-id $TENANT_ID --client-id f2b3116e-b4c4-464f-80ec-73cd9d9886b4
 ```
+
+## Secrets
+
+Spice integrates with multiple secret stores to help manage sensitive data securely. For detailed information on supported secret stores, refer to the [secret stores documentation](/components/secret-stores). Additionally, learn how to use referenced secrets in component parameters by visiting the [using referenced secrets guide](/components/secret-stores#using-secrets).
+
+## Cookbook
+
+- A cookbook recipe to configure Sharepoint as a data connector in Spice. [SharePoint Data Connector Recipe](https://github.com/spiceai/cookbook/tree/trunk/sharepoint#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/sharepoint.md]:
```suggestion
- A cookbook recipe to configure Sharepoint as a data connector in Spice. [SharePoint Data Connector](https://github.com/spiceai/cookbook/tree/trunk/sharepoint#readme)
```

File: spiceaidocs/docs/components/data-connectors/snowflake.md [hunk 24]
```diff
@@ -193,3 +209,11 @@ datasets:
...
 
 :::
+
+## Secrets
+
+Spice integrates with multiple secret stores to help manage sensitive data securely. For detailed information on supported secret stores, refer to the [secret stores documentation](/components/secret-stores). Additionally, learn how to use referenced secrets in component parameters by visiting the [using referenced secrets guide](/components/secret-stores#using-secrets).
+
+## Cookbook
+
+- A cookbook recipe to configure Snowflake as a data connector in Spice. [Snowflake Data Connector Recipe](https://github.com/spiceai/cookbook/tree/trunk/snowflake#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/snowflake.md]:
```suggestion
- A cookbook recipe to configure Snowflake as a data connector in Spice. [Snowflake Data Connector](https://github.com/spiceai/cookbook/tree/trunk/snowflake#readme)
```

File: spiceaidocs/docs/components/data-connectors/spark.md [hunk 25]
```diff
@@ -118,3 +118,7 @@ Check [Secrets Stores](/components/secret-stores) for more details.
 
   </TabItem>
 </Tabs>
+
+## Cookbook
+
+- A cookbook recipe to configure Spark as a data connector in Spice. [Apache Spark Data Connector Recipe](https://github.com/spiceai/cookbook/tree/trunk/spark#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/spark.md]:
```suggestion
- A cookbook recipe to configure Spark as a data connector in Spice. [Apache Spark Data Connector](https://github.com/spiceai/cookbook/tree/trunk/spark#readme)
```

File: spiceaidocs/docs/components/data-connectors/spiceai.md [hunk 26]
```diff
@@ -44,3 +44,7 @@ The Spice.ai Cloud Platform dataset URI. To query a dataset in a public Spice.ai
   acceleration:
     enabled: true
 ```
+
+## Cookbook
+
+- A cookbook recipe to configure Spice.ai Cloud Platform as a data connector in Spice. [Spice.ai Cloud Platform Data Connector Recipe](https://github.com/spiceai/cookbook/tree/trunk/spiceai#readme)
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/spiceai.md]:
```suggestion
- A cookbook recipe to configure Spice.ai Cloud Platform as a data connector in Spice. [Spice.ai Cloud Platform Data Connector](https://github.com/spiceai/cookbook/tree/trunk/spiceai#readme)
```

**Review by lukekim — APPROVED** [review]

**Pull request merged**